src/app/container/container.component.html

```diff
@@ -21,7 +21,8 @@
       </button>
       <span class="glyphicon glyphicon-warning-sign">
       </span>
-      {{ missingContent.length === 1 ? 'Field that is missing from file: ' : 'Fields that are missing from file: '}} {{missingContent}}
+      {{ missingContent.length === 1 ? 'Field that is missing from file: ' : 'Fields that are missing from file: '}}
```

Comment:
Random thought: What is the recommended best practice for localization (not that its anywhere on our roadmap, but I am curious)